docs/installation/upgrade-240.rst

```diff
@@ -0,0 +1,167 @@
+.. _installation_upgrade_240:
+
+===================================
+Upgrade details to Open Forms 2.4.0
+===================================
+
+We do our best to avoid breaking changes, but at times we cannot guarantee that your
+configuration will keep working flawlessly. Open Forms 2.4.0 is such a release - and
+the manual interventions with context are documented here.
+
+Button styling
+==============
+
+Our buttons used custom design tokens since they were implemented before the `NL Design
+System buttons`_ were much of a thing. *Most* of the tokens map cleanly, and if you
+have configured any overrides in the admin environment they should have been
+automatically migrated. However, we urge you to verify the configuration on a
+test/acceptance environment and prepare the necessary changes.
+
+However, if you have made custom CSS changes, we cannot incorporate those and you will
+have to check your customizations. You will have to manually check if those changes
+are still relevant or not. Our advices is also to take a critical look - the transition
+to community components should make it much easier to change the appearance and behaviour
+using design tokens rather than CSS overrides.
+
+.. note:: We recommend setting up your own theme rather than trying to override the
+   ``.openforms-theme``, but with the CSS precedence order it should still be possible
+   to override token values via the admin interface.
+
+.. _NL Design System buttons: https://nl-design-system.github.io/utrecht/storybook/?path=/docs/css-button-tokens--docs
+
+Reference
+---------
+
+The CSS mapping below is the one we have taken into account for our automatic migration,
+using the following logic:
+
+* if the Open Forms token (RHS) exists, extract the value and assign it to the matching
+  NL DS token (LHS)
```

Comment:
Maybe expand right/left hand side? Not sure how familiar people are with these abbreviations :thinking: